backend/supervisord.conf

```diff
@@ -0,0 +1,18 @@
+[supervisord]
+nodaemon=true
+logfile=/dev/null
+logfile_maxbytes=0
+
+[program:indexing]
+command=python danswer/background/update.py
+stdout_logfile=/var/log/supervisor/update.log
+stdout_logfile_maxbytes=0
```

Comment:
this one spits out looottta junk, should probably roll it.

The file deletion one as long as we're not logging too much junk, I think we can keep around all the files that are uploaded/extracted/deleted

Reply:
Sure, will just roll both. Don't think we need to keep everything around